insights/parsers/ifcfg.py

```diff
@@ -126,3 +127,10 @@ def bonding_mode(self):
         if m in bond_mode_map:
             return bond_mode_map[m]
         return None
+
+    @property
```

Comment:
Every time when calling this `property` the `in` checking operation in line-136 will be executed, which results in a not so good efficiency.

```return self._has_empty_line```